addons/mass_mailing/tests/test_mailing_ab_testing.py

```diff
@@ -155,3 +155,21 @@ def test_ab_testing_compare_version(self):
         compare_version_record_count = self.env['mailing.mailing'].search_count(compare_version.get('domain'))
         # Record with mailing_type equals to sms should not be included in compare version count
         self.assertEqual(compare_version_record_count, 2, 'Number of records should be according to mailing type')
+
+    @mute_logger('odoo.addons.mail.models.mail_mail')
+    @users('user_marketing')
+    def test_send_minimum_one_mail_for_ab_testing(self):
+        """ Test that it should send minimum one mail(if possible) when ab_testing_pc is zero."""
```

Comment:
Maybe make the percentage something very little that would equal to 0 participants after the computation, like "2%" ?

That way we make sure to test a "real" use case.

Reply:
Hello @awa-odoo 

Actually, the computation is done in a way to send a mail to at least one participant.

Thanks